### PR TITLE
Various little fixes

### PR DESCRIPTION
**Pull request by dasJ**
- Namespace variables
- Copy helper scripts we need
- Fix names of helper scripts
- Use default('') to prevent NullPointerExceptions

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -8,5 +8,7 @@ slapd_olc_config:
   dummy: value
 slapd_monitor_config:
   dummy: value
+slapd_frontend_config:
+  dummy: value
```
Comment (SlothOfAnarchy) [inline defaults/main.yml]: are these dummy entries really needed? Can't you create an empty dict?

File: vars/main.yml [hunk 2]
```diff
@@ -77,9 +77,9 @@ default_monitor_config:
 default_mdb_config:
   # General
   olcDatabase: "{2}mdb"
-  olcDbDirectory: "{{ db_dir | mandatory }}"
+  olcDbDirectory: "{{ slapd_mdb_dir | mandatory }}"
```
Comment (SlothOfAnarchy) [inline vars/main.yml:80]: please mention and describe mandatory vars by name in readme. Also you forgot to replace usages of `db_dir` in readme.

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (dasJ) [reply defaults/main.yml]: I cannot. I really tried

**Comment by dasJ** [comment]
@SlothOfAnarchy Rebased onto master and fixed the README

File: tasks/frontend-config.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@
   command: >
     /usr/local/bin/slapd-olc-attribute-clean "{{ slapd_ldapi_socket | urlencode() | regex_replace('/', '%2F') }}" "{{ slapd_olc_rootdn }}"
     "olcDatabase={-1}frontend,cn=config"
-    {{ default_frontend_config | combine(frontend_config | join(' ') }} olcAttributeOptions olcConcurrency olcTLSCRLCheck olcTLSVerifyClient olcTLSProtocolMin
+    {{ default_frontend_config | combine(slapd_frontend_config) | join(' ') }} olcAttributeOptions olcConcurrency olcTLSCRLCheck olcTLSVerifyClient olcTLSProtocolMin
```
Comment (SlothOfAnarchy) [inline tasks/frontend-config.yml:17]: there is another use of `frontent_config` in this file that you didn't refactor.

File: defaults/main.yml (hunk 1 above)
Comment (SlothOfAnarchy) [reply defaults/main.yml]: Can't you check if it is defined when using it? Or try applying a default filter? Or try `{}`, this works for me in another role.

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

**Comment by dasJ** [comment]
@SlothOfAnarchy Pls rereview

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**